### DISCUSSION TITLE
Get https://ghcr.io/v2/: x509: certificate signed by unknown authority

### DISCUSSION THREAD

**Discussion by david-curran-90** [community/community#26917] [Actions]
<p>I’m getting an SSL error when trying to pull an image from GitHub Container Registry</p>
<pre><code class="lang-shell">sudo docker pull ghcr.io/github/super-linter:latest
Error response from daemon: Get https://ghcr.io/v2/: x509: certificate signed by unknown authority
</code></pre>
<p>This also happens when i attempt to login to GHCR</p>
<pre><code class="lang-shell">sudo docker login https://ghcr.io
Username: schizoid90
Password:
Error response from daemon: Get https://ghcr.io/v2/: x509: certificate signed by unknown authority
</code></pre>
<p>Obviously, this is stopping me from downloading images</p>

**Comment by clarkbw** [comment]
<p>Our certificate is somewhat new which may mean that the machine you’re using needs to be updated. We haven’t seen others with this error yet.  Keep us posted if you find a solution and we’ll continue looking on our side.</p>

**Comment by david-curran-90** [comment]
<p>I’ve updated the machine and rebooted</p>
<p>Now running Ubuntu 18.04.5 but still getting the error</p>
<p>I think there is some other cause. We have a fortinet firewall in our network that seems to be filtering the <a href="http://ghcr.io" rel="nofollow noopener">ghcr.io</a> domain. I’ve raised a request to get this allowed.</p>

**Comment by david-curran-90** [comment, marked as answer]
<p>Getting the filter removed seems to have done the trick. Now on to the next issue</p>
